releases/release-1.31/release-notes/maps/pr-123315-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 123315
+releasenote:
+  text: Graduated support for CDI device IDs to general availability. The `DevicePluginCDIDevices`
```

```suggestion
  text: Graduated support for Container Device Interface (CDI) device IDs to general availability. The `DevicePluginCDIDevices`
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653180502